docs/fides/docs/language/resources/system.md

```diff
@@ -27,6 +27,15 @@ A human-readable description of the System.
 
 A required value to describe the type of system being modeled, examples include: Service, Application, Third Party, etc.
 
+**responsibility_role**&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_enum_
```

Comment:
`responsibility_role` seems like it could use a naming enhancement, open to ideas here. I believe it is factual but possibly vague or not as explicit as it could be

Reply:
`data_responsibility_title` maybe? this one is tricky but I agree it can be a bit more descriptive

Reply:
I like being clearer here better for sure, thanks 👍🏽